kore/src/Kore/AST/ApplicativeKore.hs

```diff
@@ -0,0 +1,87 @@
+{-|
+Module      : Kore.AST.ApplicativeKore
+Description : TBA.
+Copyright   : (c) Runtime Verification, 2018
+-}
+module Kore.AST.ApplicativeKore
+    ( completeDefinition
+     )
```

Comment:
The closing parenthesis should line up with the opening parenthesis, and `where` should be on the same line.